tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -862,9 +862,7 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
                     args.pop("use_legacy_frontend")
                     return convert_pytorch_via_onnx(args, example_inputs, cli_parser, framework, _convert)
 
-                decoder = get_pytorch_decoder(args['input_model'], parse_input_shapes(args), example_inputs)
-                args['input_model'] = decoder
-                args["framework"] = "pytorch"
+                get_pytorch_decoder(args['input_model'], parse_input_shapes(args), example_inputs, args)
```

Comment:
decoder is represent input model for pytorch frontend, it will be misalignment with other pipelines. Aslo it is ok for us to work before parameters parsing as we make some modification in arguments